services/static-webserver/client/source/resource/osparc/ui_config.json

```diff
@@ -156,6 +156,11 @@
         "title": "${replace_me_product_name}",
         "newStudyLabel": "New Project",
         "idToWidget": "startS4LButton"
+      }, {
+        "resourceType": "service",
+        "expectedKey": "simcore/services/dynamic/s4l-ui-framework",
```

Comment:
what is this service doing? how does it differ from s4l-lite?

Reply:
This is the sim4life that supports plugins